db.go

```diff
@@ -79,6 +81,16 @@ func Open(dirname string) (*DB, error) {
 	return db, nil
 }
 
+func (db *DB) Close() {
+	db.flushingChan <- db.memtables.mutable
```

Comment:
### Prerequisite Checks
Ensure that `db.flushingChan` and `db.memtables.mutable` are not nil before sending to `db.flushingChan` and assigning a new `Memtable` to `db.memtables.mutable`:
```go
if db.flushingChan != nil && db.memtables.mutable != nil {
    db.flushingChan <- db.memtables.mutable
    db.memtables.mutable = NewMemtable(memtableSizeLimitBytes)
}
```